website/content/en/docs/provisioner.md

```diff
@@ -148,6 +156,21 @@ spec:
     clusterDNS: ["10.0.1.100"]
 ```
 
+## spec.limits
+
+The provisioner spec includes a limits section (`spec.limits.resources`), which constrains the maximum amount of resources that the provisioner will manage. 
+
+Presently, Karpenter supports `memory` and `cpu` limits. 
+
+CPU limits are described with a `DecimalSI` value, usually a natural integer. 
+
+Memory limits are described with a [`BinarySI` value, such as 1000Gi.](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
+
+Karpenter stops allocating resources once at least one resource limit is met/exceeded.
```

Comment:
We need a note like:
"Warning: Karpenter provisioning is highly parallel. Because of this, limit checking is eventually consistent, which can result in overrun during rapid scale outs.

Reply:
this is an excellent point.